src/applications/vaos/appointment-list/components/AppointmentsPage/UpcomingAppointmentLayout.jsx

```diff
@@ -105,7 +100,6 @@ export default function UpcomingAppointmentLayout({
           id={appt.id}
           className={classNames(
             'small-screen:vads-u-border-top--0',
-            'vaos-appts__listItem',
```

Comment:
Removed this style `vaos-appts__listItem` since the style name doesn't exist in any of the css pages